source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRBaseCardElementRenderer.mm

```diff
@@ -0,0 +1,31 @@
+//
+//  ACRBaseCardElementRenderer
+//  ACRBaseCardElementRenderer.h
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import "ACRBaseCardElementRenderer.h"
+
+@implementation ACRBaseCardElementRenderer
```

Comment:
>@implementation [](start = 0, length = 15)

Three methods in this file and three different bracing styles. Please pick a consistent style (preferably one shared by the whole project).